reactive/src/main/scala/colibri/reactive/Reactive.scala

```diff
@@ -0,0 +1,108 @@
+    def foreach(f: A => Unit): Cancelable = observable.unsafeForeach(f)
+    def head: Future[A]                   = observable.headIO.unsafeToFuture()(IORuntime.global)
+  }
+
+  def map[B](f: A => B): Rx[B]                           = transform(_.map(f))
+  def mapIterable[B](f: A => Iterable[B]): Rx[B]         = transform(_.mapIterable(f))
+  def mapEffect[F[_]: RunEffect, B](f: A => F[B]): Rx[B] = transform(_.mapEffect(f))
+  def mapFuture[B](f: A => Future[B]): Rx[B]             = transform(_.mapFuture(f))
+
+  def as[B](value: B): Rx[B]                           = transform(_.as(value))
+  def asDelay[B](value: => B): Rx[B]                   = transform(_.asDelay(value))
+  def asEffect[F[_]: RunEffect, B](value: F[B]): Rx[B] = transform(_.asEffect(value))
+  def asFuture[B](value: => Future[B]): Rx[B]          = transform(_.asFuture(value))
+
+  def collect[B](f: PartialFunction[A, B]): Rx[B] = transform(_.collect(f))
+  def filter(f: A => Boolean): Rx[A]              = transform(_.filter(f))
+
+  def switchMap[B](f: A => Rx[B]): Rx[B] = transform(_.switchMap(f andThen (_.observable)))
+  def mergeMap[B](f: A => Rx[B]): Rx[B]  = transform(_.mergeMap(f andThen (_.observable)))
+
+  @inline def transform[B](f: Observable[A] => Observable[B]): Rx[B] = Rx.observable(f(observable))
+}
+
+object Rx {
+  def apply[A](value: A): Rx[A]                       = observable(Observable(value))
+  def empty: Rx[Nothing]                              = observable(Observable.empty)
+  def observable[A](observable: Observable[A]): Rx[A] = new RxObservable(observable)
+
+  implicit object source extends Source[Rx] {
+    @inline def unsafeSubscribe[A](source: Rx[A])(sink: Observer[A]): Cancelable = source.observable.unsafeSubscribe(sink)
+  }
+
+  implicit object liftSource extends LiftSource[Rx] {
+    @inline def lift[H[_]: Source, A](source: H[A]): Rx[A] = Rx.observable(Observable.lift[H, A](source))
+  }
+}
+
+trait RxWriter[-A] {
+  def set(value: A): Unit
+}
+
+object RxWriter {
+  def observer[A](observer: Observer[A]): RxWriter[A] = new RxWriterObserver(observer)
+
+  implicit object sink extends Sink[RxWriter] {
+    @inline def unsafeOnNext[A](sink: RxWriter[A])(value: A): Unit          = sink.set(value)
+    @inline def unsafeOnError[A](sink: RxWriter[A])(error: Throwable): Unit = UnhandledErrorReporter.errorSubject.unsafeOnError(error)
+  }
+
+  implicit object liftSink extends LiftSink[RxWriter] {
+    @inline def lift[G[_]: Sink, A](sink: G[A]): RxWriter[A] = RxWriter.observer(Observer.lift(sink))
+  }
+}
+
+trait Var[A] extends Rx[A] with RxWriter[A]
+
+object Var {
+  def apply[A](seed: A): Var[A] = new BehaviorVar(seed)
+  def apply[A](): Var[A]        = new ReplayLatestVar[A]()
+}
+
+private final class RxObservable[A](inner: Observable[A]) extends Rx[A] {
+  private val state = new ReplayLatestSubject[A]()
+
+  val observable: Observable[A] = inner.multicast(state).refCount.distinctOnEquals
+
+  def now(): Option[A] = state.now()
```

Comment:
Now is a bit dangerous. Because the observable is not hot. If nobody is connect, now becomes outdated.